data/gegno/ships.txt

```diff
@@ -0,0 +1,954 @@
+
+ship Tridymite
+	sprite ship/tridymite
+	thumbnail thumbnail/tridymite
+	swizzle 0
+	attributes
+		category "Light Warship"
+		licenses
+			"Gegno Civilian"
+			"Scin Architect"
+			"Scin Hoplologist"
+		cost 1960000
+		mass 225
+		bunks 20
+		"cargo space" 40
+		drag 4
+		"engine capacity" 86
+		"fuel capacity" 400
+		"heat dissipation" 0.7
+		hull 5890
+		"outfit space" 190
+		"required crew" 9
+		shields 0
+		"weapon capacity" 62
+		weapon
+			"blast radius" 35
+			"shield damage" 220
+			"hull damage" 430
+			"hit force" 410
+	outfits
+		"C7 Brawl Cell"
+		"Guile Anti-Missile"
+		Hyperdrive
+		"R02 Battlezone Battery"
+		"SC-12 Plasma Thruster"
+		"SC-14 Plasma Steering"
+	engine 38.5 73
+		zoom 0.9
+	engine -38.5 73
+		zoom 0.9
+	engine 0 67.5
+	turret 0 18.5 "Guile Anti-Missile"
+	leak leak 50 40
+	leak flame 40 70
+	explode "medium explosion" 3
+	explode "small explosion" 19
+	explode "tiny explosion" 14
+	"final explode" "final explosion small"
+	description "Using a similar yet smaller frame to the Coesite, the Tridymite was designed as the production platform for the Gegno Scin's new Guile system. As its younger sibling, the Tridymite is a smaller escort vessel that provides anti-cannon screening for Scin fleets, while Coesites are armed with more offensive weaponry."
+
+ship Felsic
+	sprite ship/felsic
+	thumbnail thumbnail/felsic
+	swizzle 0
+	attributes
+		category "Light Freighter"
+		licenses
+			"Gegno Civilian"
+			"Scin Architect"
+			"Scin Hoplologist"
+		cost 2342000
+		mass 565
+		bunks 78
+		"cargo space" 160
+		drag 5.9
+		"engine capacity" 106
+		"fuel capacity" 500
+		"heat dissipation" 0.7
+		hull 11000
+		"outfit space" 310
+		"required crew" 38
+		shields 0
+		"weapon capacity" 84
+		weapon
+			"blast radius" 74
+			"shield damage" 340
+			"hull damage" 650
+			"hit force" 690
+	outfits
+		"Astuit Battery" 2
+		"C17 Warzone Core"
+		"Guile Anti-Missile"
+		Hyperdrive
+		"R02 Battlezone Battery"
+		"SC-14 Plasma Steering"
+		"SC-14 Plasma Thruster"
+	engine -18 152
+	engine 18 152
+	turret 0 -76 "Guile Anti-Missile"
+	turret -44 85.5 "Astuit Battery"
+	turret 44 85.5 "Astuit Battery"
+	leak leak 60 50
+	leak flame 60 80
+	explode "large explosion" 7
+	explode "medium explosion" 20
+	explode "small explosion" 28
+	explode "tiny explosion" 16
+	"final explode" "final explosion medium"
+	description "Felsics are essentially stripped Schists repurposed for a more engaging role. In place of the heavier engines that the Vi had mounted on, they expanded the profile have more efficient engines, dedicating the remaining space to a larger cargo capacity. On top of fitting more cargo room, the Scin also oufitted these 3 turret mounts, allowing them to double as a secondary combat vessel to support their war efforts, and to fill in the lack of a dedicated warship."
```

Comment:
```suggestion
	description "Felsics are based on Schists, but offer many upgrades in exchange for less durability and space for engines. The Scin have opted for three turret mounts, as well as expanded capacity for outfits and weapons. All this allows these ships to perform logistical duties well as well as doubling as a secondary combat vessel, making up for the lack off a dedicated Scin warship."
```
The engine capacity should probably be reduced, 85 maybe?

Reply:
Sounds good.